history/characters/42000_high_elf.txt

```diff
@@ -2543,4 +2543,65 @@
 		culture = forsaken
 	}
 	616.2.11={ death=yes }
-}
\ No newline at end of file
+}
+
+#dynasty=20027
+#Seliph Dawnweaver
+42312={
+	name=Seliph
+	dynasty=20027
+	dna="kiionckbdlg"
+	culture=high_elf religion=cult_of_sunwell
+	martial=4 diplomacy=6 stewardship=5 intrigue=4 learning=6
+	add_trait=thrifty_clerk
+	124.3.23={ birth=yes add_trait=creature_high_elf immortal_age=18 }
+	532.1.11={ death=yes }
+}
+
+#Koltira Dawnweaver AKA Deathweaver
+42313={
+	name=Koltira
+	dynasty=20027
+	father=42312
+	dna="djggkcgcenh"
+	culture=high_elf religion=cult_of_sunwell
+	martial=6 diplomacy=4 stewardship=4 intrigue=5 learning=5
+	add_trait=brilliant_strategist add_trait=sturdy add_trait=brave add_trait=proud
+	421.1.24={ birth=yes add_trait=creature_high_elf immortal_age=18 }
+	439.1.24={
+		add_trait = class_warrior_4
+	}
+	# Becomes a death knight and a blood elf
+	603.9.14={
+		remove_trait=creature_high_elf
+		remove_trait=class_warrior_4
+		add_trait=creature_blood_elf
+		add_trait=class_death_knight_5
+		culture=blood_elf
+		add_trait=being_undead
+		religion=death_god
+		employer = 60003 # Arthas
+	}
+	# Befriends Thassarian
+	604.1.1={
+		add_friend = 61197
```

Comment:
Doesn't work because it should be inside `effect`.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79923154-04b88480-8446-11ea-9190-d219780c378e.png)

</details>

```suggestion
		effect = {
			add_friend = 61197
		}
```